### DISCUSSION TITLE
Update error: cannot convert undefined or null to object

### DISCUSSION THREAD

**Discussion by techbridgedev** [facebook/docusaurus#9332] [Q&A]
Tried to update to from 2.4.1 to Docusaurus 2.4.3 with this command:

`npm i @docusaurus/core@latest @docusaurus/preset-classic@latest module-type-aliases@latest`

Upgrade fails with message `npm ERR! Cannot convert undefined or null to object`

Log attached:

[2023-09-20T22_01_00_980Z-debug-0.log](https://github.com/facebook/docusaurus/files/12677960/2023-09-20T22_01_00_980Z-debug-0.log)

**Comment by slorber** [comment]
🤷‍♂️ the stacktrace doesn't look super related to Docusaurus but more related to nvm/node/npm: maybe try another version?
 
We can't do anything without a small repo.

**Comment by techbridgedev** [comment]
Thanks; I deleted the node_modules folder, edited my package.json to use the latest 
Docusaurus versions, and ran `npm i`; it installed correctly.
